src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java

```diff
@@ -202,6 +207,14 @@ public void fromJson(JSONObject json) {
             }
         }
 
+        if (json.containsKey(HASHTAGS)) {
+            hashtags = new ArrayList<String>();
+            JSONArray tags = json.getJSONArray(HASHTAGS);
+            for (int i = 0; i < tags.size(); i++) {
+                hashtags.add(tags.getString(i));
+            }
+        }
```

Comment:
The initialization of the `ArrayList` can be done a bit more efficient. I think it is also wise to have `hashtags` be an empty list instead of `null`.
```suggestion
        if (json.containsKey(HASHTAGS)) {
            JSONArray tags = json.getJSONArray(HASHTAGS);
            hashtags = new ArrayList<String>(tags.size());            
            for (int i = 0; i < tags.size(); i++) {
                hashtags.add(tags.getString(i));
            }
        } else {
            hashtags = Collections.emptyList();
        }
```

Reply:
Done